src/Http/Middleware/ContentRoleMiddleware.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Portable\FilaCms\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Portable\FilaCms\FilaCms;
+use Illuminate\Support\Str;
+
+class ContentRoleMiddleware
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Closure(\Illuminate\Http\Request): (\Symfony\Component\HttpFoundation\Response)  $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        $contentModels = (new FilaCms())->getRawContentModels();
+
+        $path = $request->path();
```

Comment:
There's still no way for the user to create content that _everyone_ can view?  I don't see a codepath here where if the user isn't authenticated, they can see a page.

Reply:
Line 33-35. If there's no role associated with the resource.